tests/foreman/cli/test_remoteexecution.py

```diff
@@ -1307,7 +1315,9 @@ def test_positive_run_job_on_host_registered_to_pull_provider(
                 'effective-user': f'{username}',
             }
         )
-        assert_job_invocation_result(invocation_command['id'], rhel_contenthost.hostname)
+        assert_job_invocation_result(
+            target_sat, invocation_command['id'], rhel_contenthost.hostname
```

Comment:
@pondrejk `test_positive_run_job_on_host_registered_to_pull_provider` test is using both `module_targe_sat` and `target_sat` is this intended?

Reply:
ugh, god no, good chatch :) this was also introduced by 11544